test/fast/FastFrontendFacet.test.ts

```diff
@@ -25,36 +25,21 @@ interface FastFixtureOpts {
   isSemiPublic: boolean;
 }
 
-// Interfaces for returned structs.
-
-interface Details {
-  addr: string;
-  name: string;
-  symbol: string;
-  decimals: BigNumber;
-  totalSupply: BigNumber;
-  transferCredits: BigNumber;
-  isSemiPublic: boolean;
-  hasFixedSupply: boolean;
-  reserveBalance: BigNumber;
-  memberCount: BigNumber;
-  governorCount: BigNumber;
-}
-
-interface MemberDetails {
-  addr: string;
-  balance: BigNumber;
-  ethBalance: BigNumber;
-  isGovernor: boolean;
+// A way, not the best way, to get a POJO from a struct.
+const structToObj = (struct: {}) => {
```

Comment:
Since this is discarding half of the (repeating) values, I would rename the function to make sure it's clear.
Such as `chainStructToObj` - and maybe put it in our `utils` so we can reuse it?